### PR TITLE
add readme in pose2twist

### PR DESCRIPTION
**Pull request by TakaHoribe**
Signed-off-by: Takamasa Horibe <horibe.takamasa@gmail.com>

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: localization/twist_estimator/pose2twist/src/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# pose2twist
+
+## Purpose
+
+This `pose2twist` calculates the velocity from the input pose history. In addition to the computed twist, this node outputs the linear-x and angular-z components as a float message to simplify debugging.
+
+The `twist.linear.x` is calculated as `sqrt(dx * dx + dy * dy + dz * dz) / dt`, and the values in the `y` and `z` field are zero.
```
Comment (kmiya) [inline localization/twist_estimator/pose2twist/src/README.md:7]:
```suggestion
The `twist.linear.x` is calculated as `sqrt(dx * dx + dy * dy + dz * dz) / dt`, and the values in the `y` and `z` fields are zero.
```
Nit: fields? (not sure)

**Review by taikitanaka3 — APPROVED** [review]
LGTM

File: localization/twist_estimator/pose2twist/src/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+The `twist.angular` is calculated as `d_roll / dt`, `d_pitch / dt` and `d_yaw / dt` for each field.
+
+## Inputs / Outputs
+
+### Input
+
+| Name | Type                            | Description                                      |
+| ---- | ------------------------------- | ------------------------------------------------ |
+| pose | geometry_msgs::msg::PoseStamped | pose source to used for the velocity calculation |
```
Comment (kmiya) [inline localization/twist_estimator/pose2twist/src/README.md:16]: IMO: Might need to add a period to Description, or remove periods in the other table.
This rule is not unified in all READMEs, so not sure which would be better.

Comment (TakaHoribe) [reply localization/twist_estimator/pose2twist/src/README.md:16]: Thank you. I added it.

**Comment by TakaHoribe** [comment]
@YamatoAndo マージします！

File: localization/twist_estimator/pose2twist/src/README.md (hunk 2 above)
Comment (kmiya) [reply localization/twist_estimator/pose2twist/src/README.md:16]: Not be fixed yet?

Comment (TakaHoribe) [reply localization/twist_estimator/pose2twist/src/README.md:16]: Ohh, sorry. I'll create a new PR.

Comment (kmiya) [reply localization/twist_estimator/pose2twist/src/README.md:16]: Thanks!!

**Pull request merged**